test/parallel/test-tls-passphrase.js

```diff
@@ -49,177 +49,195 @@ const server = tls.Server({
-  }, common.mustCall());
-
-  // string
-  tls.connect({
-    port: this.address().port,
-    key: passKey.toString(),
-    passphrase: 'passphrase',
-    cert: cert.toString(),
-    rejectUnauthorized: false
-  }, common.mustCall());
-
-  tls.connect({
-    port: this.address().port,
-    key: rawKey.toString(),
-    cert: cert.toString(),
-    rejectUnauthorized: false
-  }, common.mustCall());
-
-  tls.connect({
-    port: this.address().port,
-    key: rawKey.toString(),
-    passphrase: 'ignored',
-    cert: cert.toString(),
-    rejectUnauthorized: false
-  }, common.mustCall());
-
-  // String[]
-  tls.connect({
-    port: this.address().port,
-    key: [passKey.toString()],
-    passphrase: 'passphrase',
-    cert: [cert.toString()],
-    rejectUnauthorized: false
-  }, common.mustCall());
-
-  tls.connect({
-    port: this.address().port,
-    key: [rawKey.toString()],
-    cert: [cert.toString()],
-    rejectUnauthorized: false
-  }, common.mustCall());
-
-  tls.connect({
-    port: this.address().port,
-    key: [rawKey.toString()],
-    passphrase: 'ignored',
-    cert: [cert.toString()],
-    rejectUnauthorized: false
-  }, common.mustCall());
-
-  // Object[]
-  tls.connect({
-    port: this.address().port,
-    key: [{ pem: passKey, passphrase: 'passphrase' }],
-    cert: cert,
-    rejectUnauthorized: false
-  }, common.mustCall());
-
-  tls.connect({
-    port: this.address().port,
-    key: [{ pem: passKey, passphrase: 'passphrase' }],
-    passphrase: 'ignored',
-    cert: cert,
-    rejectUnauthorized: false
-  }, common.mustCall());
-
-  tls.connect({
-    port: this.address().port,
-    key: [{ pem: passKey }],
-    passphrase: 'passphrase',
-    cert: cert,
-    rejectUnauthorized: false
-  }, common.mustCall());
-
-  tls.connect({
-    port: this.address().port,
-    key: [{ pem: passKey.toString(), passphrase: 'passphrase' }],
-    cert: cert,
-    rejectUnauthorized: false
-  }, common.mustCall());
-
-  tls.connect({
-    port: this.address().port,
-    key: [{ pem: rawKey, passphrase: 'ignored' }],
-    cert: cert,
-    rejectUnauthorized: false
-  }, common.mustCall());
-
-  tls.connect({
-    port: this.address().port,
-    key: [{ pem: rawKey.toString(), passphrase: 'ignored' }],
-    cert: cert,
-    rejectUnauthorized: false
-  }, common.mustCall());
-
-  tls.connect({
-    port: this.address().port,
-    key: [{ pem: rawKey }],
-    passphrase: 'ignored',
-    cert: cert,
-    rejectUnauthorized: false
-  }, common.mustCall());
-
-  tls.connect({
-    port: this.address().port,
-    key: [{ pem: rawKey.toString() }],
-    passphrase: 'ignored',
-    cert: cert,
-    rejectUnauthorized: false
-  }, common.mustCall());
-
-  tls.connect({
-    port: this.address().port,
-    key: [{ pem: rawKey }],
-    cert: cert,
-    rejectUnauthorized: false
-  }, common.mustCall());
+  function test(options) {
+    const next = testCases.pop();
+    if (next) {
+      test(next);
+    } else {
+      server.close();
+    }
```

Comment:
It doesn't look like the rewritten test ever calls `tls.connect()`.

Reply:
🤦‍♂ That's why it's so much faster.  😆  I had it in there but refactored it out. Ooff...

Reply:
And....fixing it means the test isn't anymore reliable...back to the drawing board....